src/entities/models/dao.py

```diff
@@ -85,7 +85,7 @@ class FilingDAO(Base):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     filing_period: Mapped[str] = mapped_column(ForeignKey("filing_period.code"))
     lei: Mapped[str]
-    tasks: Mapped[List[FilingTaskProgressDAO]] = relationship(lazy="selectin", cascade="all, delete-orphan")
+    tasks: Mapped[Optional[List[FilingTaskProgressDAO]]] = relationship(lazy="selectin", cascade="all, delete-orphan")
```

Comment:
does `Mapped[List[...] | None]` not work for this?

Reply:
Gah I think this is the second time I've done this.  I need to scrape Optional from my brain in favor of None